client/components/App/index.js

```diff
@@ -0,0 +1,3 @@
+import App from './App';
```

Comment:
I get you, your point is definitely valid.

The reason I like it is that having lots of different index.jsx files open feels disorienting to me - it's hard to tell them apart. It also makes opening files with a command pallet, i.e. `command + shift + p` a lot easier since you can type `App.jsx` to find the app component instead of `app/index.jsx`.

Another benefit is that the index.js can be a way to documents the exports of the folder. Often a folder will export more than just the main component inside it.

So for me I think the extra file is worth it - but if you're still not convinced ....... I'm happy to use index.jsx.

Reply:
> The reason I like it is that having lots of different index.jsx files open feels disorienting to me - it's hard to tell them apart. It also makes opening files with a command pallet, i.e. `command + shift + p` a lot easier since you can type `App.jsx` to find the app component instead of `app/index.jsx`.

Well I'm certainly in the group to not mind having to do `App/index.jsx` haha, but I definitely understand that's harder

> Another benefit is that the index.js can be a way to documents the exports of the folder. Often a folder will export more than just the main component inside it.

Well I like this point for the future thinking so I'm sold.

Reply:
Woo cool 😸